parts/k8s/kubernetesagentvars.t

```diff
@@ -3,23 +3,15 @@
     "{{.Name}}StorageAccountsCount": "[add(div(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')), mod(add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),2), add(mod(variables('{{.Name}}Count'), variables('maxVMsPerStorageAccount')),1)))]",
 {{end}}
     "{{.Name}}Count": "[parameters('{{.Name}}Count')]",
+    "{{.Name}}VMNamePrefix": "{{GetAgentVMPrefix .}}",
```

Comment:
I am guessing this needs to be

`"{{.Name}}VMNamePrefix": "{{GetAgentVMPrefix .Name}}"`

Reply:
No, the function still takes an agent profile

Reply:
`getAgentPoolIndexByName` I've changed to just take in the name though

Reply:
Ah yes. I see that now. Makes sense.